### PR TITLE
Fix typo in readme file

### PR DESCRIPTION
**Pull request by dhananjaym2**
1. Fix spelling mistake in readme file for java code.
2. Complete the code when someone wants to directly copy and paste the code from the README file in their own source code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,16 +62,23 @@ android {
 ```
 
 ## Starting the PhotoEditor activity
-Simply add this line to the place from where you want to start the activity-
 
+In the activity from where you want to start the photo editor add this line outside all methods in your activity class-
 ```java
+final int PHOTO_EDITOR_REQUEST_CODE = 231;// Any integer value as a request code.
```
Comment (iamutkarshtiwari) [inline README.md]: `private static final int PHOTO_EDITOR_REQUEST_CODE = 231;`

File: README.md [hunk 2]
```diff
@@ -62,16 +62,23 @@ android {
...
+
+ try {
   Intent intent = new ImageEditorIntentBuilder(this, sourceImagePath, outputFilePath)
          .withAddText() // Add the features you need 
          .withPaintFeature()
          .withFilterFeature()
          .withRotateFeature()
          .withCropFeature()
-         .withBrighnessFeature()
+         .withBrightnessFeature()
```
Comment (iamutkarshtiwari) [inline README.md]: Cool

File: README.md [hunk 3]
```diff
@@ -62,16 +62,23 @@ android {
...
 
 ## Starting the PhotoEditor activity
-Simply add this line to the place from where you want to start the activity-
 
+In the activity from where you want to start the photo editor add this line outside all methods in your activity class-
 ```java
+final int PHOTO_EDITOR_REQUEST_CODE = 231;// Any integer value as a request code.
+```
+
+Simply add this line in a method from where you want to start the Ananas Photo Editor activity-
```
Comment (iamutkarshtiwari) [inline README.md]: `Use the following code to build and launch the photo editor:`

File: README.md [hunk 4]
```diff
@@ -62,16 +62,23 @@ android {
 ```
 
 ## Starting the PhotoEditor activity
-Simply add this line to the place from where you want to start the activity-
 
+In the activity from where you want to start the photo editor add this line outside all methods in your activity class-
```
Comment (iamutkarshtiwari) [inline README.md]: `Add this global static constant to your activity class with your preferred request code:`

File: README.md (hunk 1 above)
Comment (dhananjaym2) [reply README.md]: Here static is not required, since we're accessing the same variable in the same class. private is fine.

**Pull request merged**